Overview.bs

```diff
@@ -575,29 +575,69 @@ parts of some content an incremental font can render:
 
 * Any shaping units that passed both checks can be rendered in their entirety with the font.
 
-The client may also wish to know what the font can render at a more granular level than a shaping unit. The following
-procedure can produce an approximate check at the Unicode code point level for any shaping units which failed the previous check:
-
-* For each code point within a shaping unit:
-
-    *  If there is not a [[open-type/cmap|cmap]] entry for the code point, or the entry maps to glyph 0 then the incremental font does not
-         support rendering that code point.
-
-    *  Compute the point in design space and set of layout features which will apply to that
-         code point. Form a [=font subset definition=] which contains the code point, computed design space point, and layout features.
-         Execute the [$Extend an Incremental Font Subset$] algorithm, stopping at step 6. If the entry list is not empty then the
-         incremental font does not fully support rendering this specific code point at this position in the text.
+The client may also wish to know what the font can render at a more granular level than a shaping unit. The following pseudo code
+demonstrates a possible method for splitting a shaping unit which failed the above check up into spans which can be rendered using the
+incremental font:
+
+<pre highlight="python">
+# Returns a list of spans, [start, end] inclusive, within shaping_unit that are
+# supported by and can be safely rendered with ift_font.
+#
+# shaping_unit is an array where each item has a code point, associated list of
+# features, and the design space point that code point is being rendered with.
+def supported_spans(shaping_unit, ift_font):
+  current_start = None
```

Comment:
This and the below are pseudo-code cases where I like chaining such assignments on one line, I guess based on a lines-are-emphasis way of thinking. But YMMV.

Reply:
Sounds good, fixed